tests/test_spark.py

```diff
@@ -187,15 +187,15 @@ def test_s3():
     utils.run_tests(app_url=utils._scala_test_jar_url(),
                     app_args=app_args,
                     expected_output="Read 3 lines",
-                    app_name="/spark",
                     args=args)
 
 
+@pytest.mark.xfail(utils.is_strict(), reason="Currently fails in strict mode")
```

Comment:
We should change this test to something that can be exercised in `strict`, probably still good to test this as folding (marathon groups) is becoming the norm.

Reply:
Yeah, guess it doesn't matter much which test it is. I'll change it to SparkPi.